linode/manager
# M3-5603: Clean up GooglePay code

## Description
Cleans up some Google Pay code

## How to test
Go to `account/billing` and ensure that the following still works:
- Adding GooglePay as a recurring payment method
- Making a one time payment with the GooglePay Button
- Error handling (toasts and notices)
